pkg/proxy/nftables/proxier.go

```diff
@@ -193,6 +194,13 @@ type Proxier struct {
 	serviceCIDRs string
 
 	logger klog.Logger
+
+	clusterIPs          *nftContainer[[1]string, [0]string]
+	serviceIPs          *nftContainer[[3]string, [1]string]
+	firewallIPs         *nftContainer[[3]string, [1]string]
+	noEndpointServices  *nftContainer[[3]string, [1]string]
+	noEndpointNodePorts *nftContainer[[2]string, [1]string]
+	serviceNodePorts    *nftContainer[[2]string, [1]string]
```

Comment:
:raised_eyebrow: 
We're going to need nicer types than that...

Reply:
any suggestions?